paddle/fluid/pybind/inference_api.cc

```diff
@@ -44,6 +45,7 @@ static void BindNativeConfig(py::module *m);
 static void BindNativePredictor(py::module *m);
 static void BindAnalysisConfig(py::module *m);
 static void BindAnalysisPredictor(py::module *m);
+static void BindMkldnnQuantizerConfig(py::module *m);
```

Comment:
Please, wrap this line with `#ifdef PADDLE_WITH_MKLDNN`

Reply:
Done